sharedPrompts/agent-tools/update_postman.py

```
@@ -0,0 +1,104 @@
+            elif item.get('request', {}).get('url', {}).get('raw', '').endswith('/prompts/generate'):
+                body = item.get('request', {}).get('body', {}).get('raw', '')
+                if body:
+                    try:
+                        parsed = json.loads(body)
+                        if 'request_type' in parsed and 'variant' not in parsed:
+                            parsed['variant'] = None
+                            # formatting back
+                            item['request']['body']['raw'] = json.dumps(parsed, indent=2, ensure_ascii=False)
+                            updated_variant += 1
+                    except Exception:
+                        pass
```

Ruff (0.15.4)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)